sdk/communication/azure-communication-email/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0b1 (TODO: UPDATE WITH RELEASE DATE)
```

Comment:
We usually just use `"## 1.0.0b1 (Unreleased)"` in main until we know what date it will be shipped.